docs/reference/document_tree.rst

```diff
@@ -4,33 +4,30 @@ Document Tree
 This admin integrates with the `CmfTreeBrowserBundle`_ to provide a tree view
 of the documents.
 
-Enable the Bundles
-------------------
+Download the Bundle
+-------------------
```

Comment:
this is just an empty title... did you mean to add a `composer require` here?

Reply:
Done